### ISSUE TITLE
Zeolite Ore not picked up by Visual Prospecting on Journeymap

### ISSUE THREAD

**Issue by dermingo** [GTNewHorizons/GT-New-Horizons-Modpack#9176]
### Your GTNH Discord Username

dermingo#4429

### Your Pack Version

2.1.2.0

### Your Server

SP

### Type of Server

Single Player

### Your Expectation

I move to an ore chunk coordinate and load my Ore Finder wand with Zeolite ore.
![image](https://user-images.githubusercontent.com/34583332/146636744-0c40f1c4-06ea-4c29-aa8e-423f719defbc.png)
![image](https://user-images.githubusercontent.com/34583332/146636749-d63c1846-1b9e-4a8b-bb29-f8f0115b29f4.png)

Wand turns completely upside down.
![image](https://user-images.githubusercontent.com/34583332/146636755-35a9b5b3-42f7-4fc3-acee-c25c76cef80d.png)

Chunk happens to have Zeolite ore; confirmed by digging down.
![image](https://user-images.githubusercontent.com/34583332/146636767-e8d4d546-4474-4f19-90f8-c67f8a887776.png)

Location is logged on JourneyMap with Visual Prospecting for Ore Mix 08 with Zeolite, according to the GTNH Spreadsheet.

### The Reality

Location is not logged on Journey Map and remains an empty square on the grid.
![image](https://user-images.githubusercontent.com/34583332/146636778-62b3920e-1388-4007-934f-e30bc6dcec73.png)

### Your Proposal

I just want to confirm if this is a bug and the location should have been logged.
Or are there any ore mixes that have been disabled?

### Final Checklist

- [X] I have searched this issue tracker and there is nothing similar already. Posting on a closed issue saying the bug still exists will prompt us to investigate and reopen it once we confirm your report.
- [x] I can reproduce this problem consistently by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.
- [ ] I have asked other people and they confirm they also have this problem by follow the exact steps I described above, or this does not need reproducing, e.g. recipe loophole.

**Comment by SinTh0r4s** [comment]
Hey, thanks for the issue! Did you play on that world before the update to 2.1.2.0? And especially the chunk: was that particular chunk already generated? Is there a lot of caves/ravines in the ore vein?

GT does not save the location of ore veins after generation. VP needs to guess them when a world is loaded first. And that algorithm will sometimes drop a few veins here and there. Newly generated veins (after VisualProspecting was added) will work 100%, because I added a hook to the ore vein generation to save that information right away.

**Comment by dermingo** [comment]
Yes, this world was created on the version before 2.1.2.0. I've explored a fair bit since then and this area is already revealed on Journeymap so the chunk should have been already generated.
I'm assuming that that should not be the issue since the chunks around it have also been generated previously. I will check if there are caves underneath but I thought, going over the Ore Gen in the wiki, this was avoided if there are too much air blocks at that y-level?
Does this mean that there is a minimum amount of ores for VP to register it as a vein? The ore finder wand flipping over is not enough?

**Comment by bombcar** [comment]
It only needs the minimum if the chunk existed before VP was added. New chunks will work correctly. 

You could chunk trim your world if you really wanted to.

**Comment by dermingo** [comment]
So I dug down and the only cave I could find is tiny:
![image](https://user-images.githubusercontent.com/34583332/146659491-40779e89-10a3-437f-9054-677479d08133.png)
My ore finder wand is still strongly indicating that there are zeolite under me even at bedrock level.
I also checked that that particular ore mix only spawns at around 50-70 y and it looks like there is a lot of dirt around this level:
![image](https://user-images.githubusercontent.com/34583332/146659554-abe34f9c-72d9-468d-9403-f0aef31366d4.png)

I want to leave chunk trimming as last resort.

**Comment by bombcar** [comment]
Ore finder wand points down even if the ore is above you. If you want to you can cheat in the electric prospector and do a scan and take a screenshot

**Comment by dermingo** [comment]
I did not know that. I guess that would make sense, having the ore finder wand point up and make the noise would be confusing.
So I didn't cheat in an electric one but I loaded a backup of my world and ran over to the location again and I guess VP got triggered successfully?
![image](https://user-images.githubusercontent.com/34583332/146660958-ab4db5f9-8860-43ed-8333-89ae2835d3fe.png)

It looks like this issue is solved for now. I'll look into other chunks and if it happens again, I'll try chunk trimming. I looked at how to use one in the meantime and it seems easy enough.

Thank you again for your help.
